examples/todo-typescript/src/client/SignupPage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import SignupForm from '@wasp/auth/forms/Signup';
+
+const SignupPage = () => {
```

Comment:
I also recommend avoiding arrow function expressions (in favor of regular function statements). Check https://github.com/wasp-lang/wasp/issues/487 for details.

**NB:**  When we're typing functions, arrow functions are the only option we have, but that's not what we're doing here (the linked issue describes what that means in detail :).